EXPERIMENTS.md

```diff
@@ -0,0 +1,29 @@
+# Experiments
+
+We frequently introduce new experimental features to the agent. You can use the `--experiment` flag to opt-in to them and test them out:
+
+```bash
+buildkite-agent start --experiment experiment1 --experiment expertiment2
+```
+
+Or uou can set them in your [agent configuration file](https://buildkite.com/docs/agent/v3/configuration) and they will apply whenever your agent runs:
```

Comment:
"you" typo here! Probably don't need "and they will apply whenever your agent runs" I reckon.

Reply:
Otherwise, looking ace! 🎉